frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java

```diff
@@ -175,17 +171,61 @@ public ChannelFuture start()
                             serverGroups.registerChannel(f.channel());
                         });
 
-        logger.info("Initialize server with: {}.", channelClass.getSimpleName());
-
         future.sync();
 
         ChannelFuture f = future.channel().closeFuture();
-        f.addListener((ChannelFutureListener) future -> logger.info("Model server stopped."));
-
-        logger.info("Listening on port: {}", port);
+        f.addListener(
+                (ChannelFutureListener)
+                        listener -> logger.info("{} model server stopped.", purpose));
         return f;
     }
 
+    /**
+     * Main Method that prepares the future for the channel and sets up the ServerBootstrap.
+     *
+     * @return A ChannelFuture object
+     * @throws InterruptedException if interrupted
+     */
+    public List<ChannelFuture> start()
+            throws InterruptedException, IOException, GeneralSecurityException {
+        stopped.set(false);
+
+        SslContext sslCtx = configManager.getSslContext();
+        URI inferenceAddress = configManager.getInferenceAddress();
```

Comment:
Better return SocketAddress instead of URI

Reply:
Per our discussion, will use URI because it also includes the protocol/scheme